src/components/MapController/MapController.css

```diff
@@ -1,12 +1,10 @@
-.selector {
-}
-
 .map {
+    position: relative;
```

Comment:
Ahhhh, so you position the map 'relative', but leave everything at 0px. Then it's technically a "positioned" element and becomes the closest positioned ancestor of the controls. Excellent; that's what I was missing. Thanks!